traefik/values.yaml

```
@@ -267,6 +267,7 @@ metrics:
 ##
 ##  enable optional CRDs for Prometheus Operator
 ##
+  #  createMetricsService: true
```

We should rename the value to something like this, it will give us more flexibility for configuration in the future
```suggestion
  #  service:
  #      enabled: true
```

---

Exactly what i thought about while implementing the other approach. I also added labels and annoations for that service, so that the least minimum is configurable